backend/src/Data/AccessRights.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Database.Data;
+
+public class AccessRights
```

Comment:
Currrently it seems impossible to have an unbound time period or no restriction with `UserOfInstitution` together with `AllowedUserCount` and/or `AllowedDatasetsPerTime`. I suppose to make that possible, all fields except for `InstitutionId` should be nullable, where `null` would mean "unrestricted".